.github/workflows/rust_ci.yml

```diff
@@ -22,7 +22,7 @@ permissions:
 jobs:
   build:
 
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-24.10
```

Comment:
Maybe 24.04 LTS?

Reply:
Think this one is the LTS version: https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2404-Readme.md

Reply:
Oh doh. Fingers ran faster than my brain